File: frontend/src/app/dto/climatedata/controlParameterSetDTO.ts
export interface ControlParameterSetDTO {
    controlParameterSetId?: number; // marked as optional with the ? so that I can use it both to send and retrieve
    tempNorm: number;
    tempTolerance: number;
    relHumidityNorm: number;
    relHumidityTolerance: number;
}

File: src/main/java/com/climate_monitoring_system/dto/climatedata/ControlParameterSetDTO.java
package com.climate_monitoring_system.dto.climatedata;

import lombok.Data;

@Data
public class ControlParameterSetDTO {
    private long controlParameterSetId;
    private float tempNorm;
    private float tempTolerance;
    private float relHumidityNorm;
    private float relHumidityTolerance;
}
